verl/workers/actor/dp_actor.py

```diff
@@ -405,6 +407,7 @@ def update_policy(self, data: DataProto):
                     model_inputs = {**micro_batch.batch, **micro_batch.non_tensor_batch}
                     response_mask = model_inputs["response_mask"]
                     old_log_prob = model_inputs["old_log_probs"]
+                    rollout_log_probs = model_inputs["rollout_log_probs"] if self.config.tis_imp_ratio_cap > 0 else None
```

Comment:
A very minor problem: what if the user specify `tis_imp_ratio_cap?` but not specify `calculate_log_probs = True`? I would suggest directly checking whether "rollout_log_probs" is in `model_inputs` to avoid any risk of KeyNotFoundError (note this is already very deep in the whole verl codebase, so it could be hard for the user to debug)